api/src/main/java/org/openmrs/scheduler/db/SchedulerDAO.java

```diff
@@ -39,6 +40,13 @@
 	 */
 	public TaskDefinition getTask(Integer taskId) throws DAOException;
 	
+	/**
+	 * @see SchedulerService#getTaskByUuid(String)
+	 * 
+	 * @throws DAOException
+	 */
+	public TaskDefinition getTaskByUuid(String uuid) throws DAOException;
```

Comment:
Do you really throw any DAOException?

Reply:
yes,also in the existing code thats how its thrown at the DAO layer